paddle/fluid/framework/ir/node.cc

```diff
@@ -38,6 +38,11 @@ std::unique_ptr<Node> CreateNodeForTest(OpDesc *op_desc) {
   return std::unique_ptr<Node>(new Node(op_desc));
 }
 
+std::unique_ptr<Node> CreateNodeForPybind(const std::string &name,
```

Comment:
Node should be created from graph. Unless it's for test. Pybind is not special

Reply:
fixed.